SparkPost/2web2ui
# SA-2207: Deleted old summary report code

### What Changed

- Remove old Summary report code
- Rename V2 functions to regular function names

### How To Test

- Verify the app works as expected
- Verify that the URLs still work as expected

### To Do
